
syntax = "proto3";

service MyService {
  rpc ProcessRequest (MyRequest) returns (MyResponse);
}

message MyRequest {
  string input_data = 1;
}

message MyResponse {
  string message = 1;
}
